siatest/renter/renter_test.go

```diff
@@ -278,6 +279,22 @@ func testRenterStreamingCache(t *testing.T, tg *siatest.TestGroup) {
 	// Grab the first of the group's renters
 	r := tg.Renters()[0]
 
+	// Testing setting downloadCacheSize for streaming
+	// Test setting it to larger than the defaultCacheSize
+	if err := r.RenterPostDownloadCacheSize(4); err != nil {
+		t.Fatal(err, "Could not set DownloadCacheSize to 4")
+	}
+
+	// Test resetting to the value of defaultDownloadCacheSize (2)
```

Comment:
This test needs to check if there are actually 4 cached chunks after setting it to 4 and after setting it back to 2 it needs to check if only 2 are cached.